src/vs/base/common/labels.ts

```diff
@@ -417,17 +417,22 @@ export function mnemonicMenuLabel(label: string, forceDisableMnemonics?: boolean
  * - Windows: Supported via & character (replace && with & and & with && for escaping)
  * -   Linux: Supported via _ character (replace && with _)
  * -   macOS: Unsupported (replace && with empty string)
+ * When forceDisableMnemonics is set, the withMnemonic and withoutMnemonic values will be equal.
  */
-export function mnemonicButtonLabel(label: string, forceDisableMnemonics?: boolean): string {
+export function mnemonicButtonLabel(label: string, forceDisableMnemonics?: boolean): { withMnemonic: string; withoutMnemonic: string } {
```

Comment:
Can we overload this method to just return `string` if `forceDisableMnemonics` is `true`

Reply:
```suggestion
export function mnemonicButtonLabel(label: string, forceDisableMnemonics?: boolean): { readonly withMnemonic: string; readonly withoutMnemonic: string } {
```